policies_intake.py

```diff
@@ -184,3 +202,10 @@ def coll_in_path_of_locked_dataset(ctx, actor, coll):
 
         # There is no lock present
         return False
+
+
+def _get_intake_group(coll):
+    """ Get the group type defining part of the entire groupname. 'grp-intake' or 'intake' """
+    parts = coll.split('/')[3].split('-')
```

Comment:
Two suggestions regarding this function:
- I think it would improve readability of other rules/functions that call this function if the name is a bit more specific, e.g. _get_intake_group_prefix or _get_intake_group_type. 
- Perhaps consider validating whether the input parameter has the expected format, and throwing a ValueError or similar if not. That would make it easier to troubleshoot any issues that involve other functions that pass incorrect values to this function.